### PR TITLE
Adding a navigation bar to my portfolio

### PR DESCRIPTION
**Pull request by shreyabarua**
Added changes to html and css files to support a navigation bar.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -72,13 +74,15 @@ <h5> June 2017 - August 2017 </h5>
       <p> You can check out more of my work on 
           <a href = "https://github.com/shreyabarua" target = "_blank">GitHub</a>.</p>
 
-      <h2> Cooking </h2>
+      <h2 id = "cooking" style = "padding-top:70px;" > Cooking </h2>
```
Comment (oydang) [inline portfolio/src/main/webapp/index.html]: Nit (personal preference): Move the style attributes for `home`, `projects`, `cooking`, `gallery`, `contact` into style.css.

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -7,3 +8,41 @@
...
+  top: 0;
+  width: 100%;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  background-color:#2d2075;    
+}
+
+.heading {
+    float:right;
```
Comment (oydang) [inline portfolio/src/main/webapp/style.css]: Extra whitespace in front of float? Does the auto-formatter fix this?

Comment (shreyabarua) [reply portfolio/src/main/webapp/style.css]: auto-formatter has stopped working for me again but I manually fixed it for now

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (shreyabarua) [reply portfolio/src/main/webapp/index.html]: okay, got it

**Review by oydang — APPROVED** [review]
Nicely done!

**Pull request merged**